tests/test_sagemaker/test_sagemaker_experiment.py

```diff
@@ -1,120 +1,137 @@
+    resp = sagemaker_client.list_experiments(NextToken=next_token)
 
     assert len(resp["ExperimentSummaries"]) == 7
 
     assert resp.get("NextToken") is None
 
 
 @mock_sagemaker
-def test_delete_experiment():
-    client = boto3.client("sagemaker", region_name=TEST_REGION_NAME)
-
-    experiment_name = "some-experiment-name"
-
-    resp = client.create_experiment(ExperimentName=experiment_name)
+def test_delete_experiment(sagemaker_client):
+    sagemaker_client.create_experiment(ExperimentName=TEST_EXPERIMENT_NAME)
 
-    resp = client.delete_experiment(ExperimentName=experiment_name)
+    resp = sagemaker_client.delete_experiment(ExperimentName=TEST_EXPERIMENT_NAME)
 
     assert resp["ResponseMetadata"]["HTTPStatusCode"] == 200
 
-    resp = client.list_experiments()
+    resp = sagemaker_client.list_experiments()
 
     assert len(resp["ExperimentSummaries"]) == 0
 
 
 @mock_sagemaker
-def test_add_tags_to_experiment():
-    client = boto3.client("sagemaker", region_name=TEST_REGION_NAME)
+def test_add_tags_to_experiment(sagemaker_client):
+    sagemaker_client.create_experiment(ExperimentName=TEST_EXPERIMENT_NAME)
 
-    experiment_name = "some-experiment-name"
-
-    resp = client.create_experiment(ExperimentName=experiment_name)
-
-    resp = client.describe_experiment(ExperimentName=experiment_name)
+    resp = sagemaker_client.describe_experiment(ExperimentName=TEST_EXPERIMENT_NAME)
 
     arn = resp["ExperimentArn"]
 
     tags = [{"Key": "name", "Value": "value"}]
 
-    client.add_tags(ResourceArn=arn, Tags=tags)
+    sagemaker_client.add_tags(ResourceArn=arn, Tags=tags)
 
     assert resp["ResponseMetadata"]["HTTPStatusCode"] == 200
 
-    resp = client.list_tags(ResourceArn=arn)
+    resp = sagemaker_client.list_tags(ResourceArn=arn)
 
     assert resp["Tags"] == tags
 
 
 @mock_sagemaker
-def test_delete_tags_to_experiment():
-    client = boto3.client("sagemaker", region_name=TEST_REGION_NAME)
-
-    experiment_name = "some-experiment-name"
+def test_delete_tags_to_experiment(sagemaker_client):
+    sagemaker_client.create_experiment(ExperimentName=TEST_EXPERIMENT_NAME)
 
-    resp = client.create_experiment(ExperimentName=experiment_name)
-
-    resp = client.describe_experiment(ExperimentName=experiment_name)
+    resp = sagemaker_client.describe_experiment(ExperimentName=TEST_EXPERIMENT_NAME)
 
     arn = resp["ExperimentArn"]
 
     tags = [{"Key": "name", "Value": "value"}]
 
-    client.add_tags(ResourceArn=arn, Tags=tags)
+    sagemaker_client.add_tags(ResourceArn=arn, Tags=tags)
 
-    client.delete_tags(ResourceArn=arn, TagKeys=[i["Key"] for i in tags])
+    sagemaker_client.delete_tags(ResourceArn=arn, TagKeys=[i["Key"] for i in tags])
 
     assert resp["ResponseMetadata"]["HTTPStatusCode"] == 200
 
-    resp = client.list_tags(ResourceArn=arn)
+    resp = sagemaker_client.list_tags(ResourceArn=arn)
 
     assert resp["Tags"] == []
+
+
+@mock_sagemaker
+def test_list_tags_experiment(sagemaker_client):
```

Comment:
I don't think this test adds anything tbh. We know that the method returns the correct results from the `test_add_tags..` and `test_delete_tags..`, and we know that the method paginates correctly because that was tested for sagemaker endpoints. Unless I'm missing something that is specific to experiments, let's just remove this.